docs/_publish/launch-process/zap-templates.md

```diff
@@ -269,3 +269,61 @@ If breaking changes exist between the previous and newly promoted integration ve
 When deprecating a version of your integration, any Zap templates using the integration will automatically be marked as _Invalid_ within 24 hours of the deprecation.
 
 While invalid, the Zap template will not be _Public_ until it is adjusted to use a non-deprecated version, re-submitted for review and approved. This ensures users won’t use deprecated and broken Zap templates to make Zaps. If you’re having trouble accessing _Invalid_ templates, please [submit a ticket.](https://developer.zapier.com/contact)
+
+## Frequenly Asked Questions
+
+**Q:** How come I can see Zap templates on my directory page that don’t appear in the developer platform?
```

Comment:
Why do I see Zap templates on my app's directory page that don’t appear under my [Zap Templates](https://developer.zapier.com/zap-templates).